docker/nginx/sites-available/osm.conf

```diff
@@ -0,0 +1,111 @@
+server {
+  listen 80;
+  listen 81;
+  server_name osm.posm.io;
+
+  proxy_buffering off;
+  proxy_http_version 1.1;
+  proxy_set_header Upgrade $http_upgrade;
+  proxy_set_header Connection 'upgrade';
+  proxy_set_header Host $host;
+  proxy_cache_bypass $http_upgrade;
+
+  client_max_body_size 100M;
+
+  # {{#auth}}
+  # auth_basic "POSM";
+  # auth_basic_user_file htpasswd;
+  # {{/auth}}
+
+  location ~ /\. {
+    # do not serve dot files ever
+    return 404;
+  }
+
+  # proxied locations
+
+  # location ~ ^/api/0.6/map$ {
+  #   include /etc/nginx/fastcgi_params;
+  #   fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  # }
+
+  # location ~ ^/api/0.6/(node|way|relation|changeset)/[0-9]+$ {
+  #   include /etc/nginx/fastcgi_params;
+
+  #   if ($request_method = GET) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+
+  #   if ($request_method = HEAD) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+  # }
+
+  # location ~ ^/api/0.6/(node|way|relation)/[0-9]+/history$ {
+  #   include /etc/nginx/fastcgi_params;
+
+  #   if ($request_method = GET) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+
+  #   if ($request_method = HEAD) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+  # }
+
+  # location ~ ^/api/0.6/(way|relation)/[0-9]+/full$ {
+  #   include /etc/nginx/fastcgi_params;
+
+  #   if ($request_method = GET) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+
+  #   if ($request_method = HEAD) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+  # }
+
+  # location ~ ^/api/0.6/(nodes|ways|relations)$ {
+  #   include /etc/nginx/fastcgi_params;
+
+  #   if ($request_method = GET) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+
+  #   if ($request_method = HEAD) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+  # }
+
+  # location ~ ^/api/0.6/changeset/[0-9]+/download$ {
+  #   include /etc/nginx/fastcgi_params;
+
+  #   if ($request_method = GET) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+
+  #   if ($request_method = HEAD) {
+  #     fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  #   }
+  # }
+
+  # som-web/public/land.html is missing required by iD (https://github.com/openstreetmap/iD/blob/develop/dist/land.html)
+  # https://github.com/openstreetmap/openstreetmap-website/issues/1760
```

Comment:
I see many lines being commented out. Are they needed? or will some kind of parsing be done on the commented lines?

Reply:
@bewakes Those are from the original Nginx config for osm-web from posm-build. My primary focus was on replay-tool so I have done minimal addition/modification for osm-web. After the replay-tool is good to go, we can start configuring the rest of the modules too.